src/App.tsx

```diff
@@ -15,36 +22,79 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+export const App: FC = () => {
```

Comment:
In short - you don't need React.FC

https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/

Reply:
yeap, but i use here only FC , without React 
Maybe i something misunderstood ?